src/main/scala/org/tensorframes/impl/datatypes.scala

```diff
@@ -126,7 +126,7 @@ private[tensorframes] sealed abstract class TensorConverter[@specialized(Double,
     empty
   }
 
-  def tensor2(): tf.Tensor
+  def tensor2(): tf.Tensor[_]
```

Comment:
You can return the `Tensor[T]` here

Reply:
good point, will do.

Reply:
Actually I don't think I can not do that, the types are not the same. The returned type is from java.lang. E.g. Tensor[java.lang.Float]

Reply:
ok never mind then. We can fix that later.